zulipterminal/helper.py

```diff
@@ -456,9 +456,7 @@ def match_stream(stream: Any, text: str) -> bool:
     False otherwise.
     """
     stream_name = stream[0].lower()
-    if stream_name.startswith(text.lower()):
-        return True
-    return False
+    return text in stream_name
```

Comment:
`text.lower()`?

Reply:
@neiljp I pass `query`, which is already in lowercase, to this function. I extracted `text[1:].lower()` as `query`.

Reply:
I did see that. Let's leave this pending until the sorting is simplified - that may clarify this.